### ISSUE TITLE
Runtime error for early stopping with FB15k-237

### ISSUE THREAD

**Issue by PeruBhardwaj** [Accenture/AmpliGraph#60]
### Description
I am trying to replicate the performance results from the documentation but getting a runtime error when using early stopping with FB15k-237. 

I have used early stopping with other datasets like FB15k, WN18 and there is no runtime error with these.

### Steps to Reproduce
<img width="1116" alt="Screen Shot 2019-03-20 at 17 15 43" src="https://user-images.githubusercontent.com/16183384/54705202-2bfe9d00-4b34-11e9-8e1c-fbe0aff669ce.png">
<img width="1127" alt="Screen Shot 2019-03-20 at 17 17 05" src="https://user-images.githubusercontent.com/16183384/54705213-31f47e00-4b34-11e9-9f28-a9e6922a9eb8.png">
<img width="1046" alt="Screen Shot 2019-03-20 at 17 18 09" src="https://user-images.githubusercontent.com/16183384/54705218-3456d800-4b34-11e9-89cf-746cfb79620e.png">

**Comment by sumitpai** [comment]
fb15k-237 has some entities in test and validation set that are not in train set. So we need to remove them before fitting/evaluation. That's why you are getting this error.

**Comment by PeruBhardwaj** [comment]
Will the **train_test_split_no_unseen()** function work for this?

**Comment by sumitpai** [comment]
Just checked. There is a clean_data method in predictive_performance.py which can be used. You can also refer that file for more details. 

I think we must move the clean_data to model_utils. I will create a separate issue for that.

**Comment by sumitpai** [comment]
Closing this issue and creating a separate issue for moving clean_data to a meaningful location
